lib/util/parseHeaders.js

```diff
@@ -0,0 +1,29 @@
+const parseEmojis = str => {
+  const emojiData = require('markdown-it-emoji/lib/data/full.json')
+  return str.replace(/:(.+?):/g, (placeholder, key) => emojiData[key] || placeholder)
+}
+
+const unescapeHtml = html => html
+  .replace(/&quot;/g, '"')
+  .replace(/&#39;/g, '\'')
+  .replace(/&#x3A;/g, ':')
+  .replace(/&lt;/g, '<')
+  .replace(/&gt;/g, '>')
+
+const removeMarkdownToken = str => str
+  .replace(/`(.*)`/, '$1')  // ``
+  .replace(/\[(.*)\]\(.*\)/, '$1')  // []()
+  .replace(/\*\*(.*)\*\*/, '$1')  // **
+  .replace(/\*(.*)\*/, '$1')  // *
+  .replace(/_(.*)_/, '$1')  // _
+
+// put here to avoid circular references
+const compose = (...processors) => {
+  if (processors.length === 0) input => input
```

Comment:
Just curious, did you miss `return` here? And the next line?

Reply:
Nice! Thanks! fixed at a8c6959